app/views/filter/_quick_filter.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="quick-filter mt-1/2">
+  <%= link_to on_fire_path, class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-onfire.svg", class: "quick-filter__icon" %>
+    On Fire
+  <% end %>
+
+  <%= link_to filter_path(sort: :favorites), class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-heart.svg", class: "quick-filter__icon" %>
+    Most Favorites
+  <% end %>
+
+  <%= link_to filter_path(sort: :views), class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-views.svg", class: "quick-filter__icon" %>
+    Most Views
+  <% end %>
+
+  <%= link_to filter_path(sort: :updated), class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-clock.svg", class: "quick-filter__icon" %>
+    Last Updated
```

Comment:
Consider adding an accessible description to these SVGs with a `title:` or `desc:`. More information can be found at https://github.com/jamesmartin/inline_svg#options.

Reply:
Good call!